Logger.Tests/FileLoggerTests.cs

```diff
@@ -1,9 +1,51 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
-
-namespace Logger.Tests;
-
-[TestClass]
-public class FileLoggerTests
-{
-    
-}
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Logger.Tests;
+
+[TestClass]
+public class FileLoggerTests
+{
+    [TestMethod]
+    public void FileLogger_IsNull_ReturnsTrue()

```

Comment:
```suggestion
    public void FileLogger_FileLoggerNotConfiguredWithPath_CreateLoggerReturnsTrue()

```
Consider a more explicit name for your tests, something like this maybe?